server/validation/signupSchema.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  number, object, string,
+} from 'yup';
+
+const signupSchema = object({
+  name: string().required('Name is required'),
+  email: string().email('Your email must be a valid email').required('Your email is required to rgister'),
+  phone: number().min(10, 'Your phone number must be 10 digits at least').required('Your phone number is required'),
```

Comment:
Try this with '12' for example and tell me if it passes or not
```suggestion
  phone: string().min(10, 'Your phone number must be 10 digits at least').required('Your phone number is required'),
```

Reply:
yes it pass because you change the type of phone in database to varchar and accept maximum 30 character